lib/renaissance_web/live/auctions_live/show.ex

```diff
@@ -42,8 +42,13 @@ defmodule RenaissanceWeb.AuctionsLive.Show do
   end
 
   def time_remaining(end_time) do
-    if Timex.before?(Timex.now(), end_time) do
-      Timex.Interval.new(from: Timex.now(), until: end_time)
+    Timex.now() |> time_remaining(end_time)
+  end
+
+  def time_remaining(start_time, end_time) do
+    # time_remaining/2 is a hack for testing
```

Comment:
😞

Reply:
I don't know of a cleaner way to test this, but suggestions are welcome!